desc/optimize/stochastic.py

```diff
@@ -0,0 +1,194 @@
+"""Function for minimizing a scalar function of multiple variables."""
```

Comment:
This looks copied from the `fmin_scalar.py` file and should probably be updated.

Reply:
That description works for both so I think it's fine, the relevant detail is in the docstring.